metrics/lib/common.bash

```diff
@@ -42,6 +42,27 @@ function save_results(){
 	bash $LIB_DIR/send_results.sh -n "$1" -a "$2" -r "$3" -u "$4"
 }
 
+# This function checks existence of commands.
+# They can be received standalone standalone or as an array, e.g.
+#
+# cmds=(“cmd1” “cmd2”)
+# check_cmds "${cmd[@]}"
+function check_cmds()
+{
+	set +e
```

Comment:
I'm not totally clear on what will happen, or how to get the desired behavior. This is most interesting: https://delx.net.au/blog/2016/03/safe-bash-scripting-set-e-is-not-enough/
In a nutshell, "set -e" often doesn't do what you want or expect, so we should probably not be counting on it in these tests at any point, and instead writing explicit checks.